File: tests/00_break_1.c
#include "glyph.h"

void t()
{
    u64 c[] = {
    };
    u16 i[] = {
        cpu_encode_op_break(0)
    };
    cpu_test("break_1", c, i);
}

int main(int argc, char **argv)
{
    t();
    return 0;
}

File: tests/00_break_1.go
package main

import . "glyph/tests/interp"

func t() {
    c := []uint64{
    }
    i := []uint16{
        CPU_encode_op_break(0),
    }
    CPU_test("break_1", c, i);
}

func main() {
    t()
}
